apps/user/models/user.go

```diff
@@ -4,20 +4,10 @@ import "github.com/jinzhu/gorm"
 
 type User struct {
 	gorm.Model
-	Email string `gorm:"type:varchar(100);unique_index"`
-	Name  string
-	Type  string
-}
-
-type Group struct {
-	gorm.Model
-	Name string
-}
-
-type Role struct {
-	gorm.Model
-	Name    string
-	UserID  uint `gorm:"index"`
-	GroupID uint `gorm:"index"`
-	Type    string
+	Email     string `gorm:"type:varchar(100); not null; unique_index"`
```

Comment:
We should have a varchar of 255 length as email could be longer.

Reply:
done